common/filetoolkitwithundo.cpp

```diff
@@ -226,19 +226,32 @@ ReturnCode FileToolkitWithUndo::createSharedMountPoint(const std::string &path)
         return ReturnCode::FAILURE;
     }
 
-    m_cleanupHandlers.push(new MountCleanUpHandler(path));
+    m_cleanupHandlers.push_back(new MountCleanUpHandler(path));
     log_debug() << "Created shared mount point at " << path;
 
     return ReturnCode::SUCCESS;
 }
 
+bool FileToolkitWithUndo::isPathExist(const std::string path)
```

Comment:
The name of this function isn't very obvious. hasPathAlready or something would make more sense.